installation/install.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Installation</title>
+  <script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js?lang=ruby&amp;skin=sons-of-obsidian">
+  </script>
+  <link rel="stylesheet" href="/css/nav.css">
+  <link rel="stylesheet" href="/css/sidebar.css">
+  <link rel="stylesheet" href="/css/components.css">
+  <link rel="stylesheet" href="/installation/install.css">
+  <link href='https://unpkg.com/boxicons@2.1.1/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+  <div class="container">
+    <nav class="nav-bar">
+      <div class="left-nav">
+        <a href="/index.html">
+          UI Book
+        </a>
+      </div>
+      <button href="#" class="toggle-btn">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </button>
+      <ul class="right-nav">
+        <li><a href="/index.html">Home</a></li>
+        <li><a href="/components/alert/alert.html">Documentation</a></li>
+        <li><a href="https://github.com/vermasavita/">GitHub</a></li>
+      </ul>
+    </nav>
+    <div class="doc-container">
+      <div class="component-menu">
+        <div class="titles"><strong>Components</strong>
+
+        </div>
+        <div class="list-items">
+          <ul>
+            <li><a href="/components/alert/alert.html">Alert</a></li>
+            <li><a href="/components/avatar/avatar.html">Avatar</a></li>
+            <li><a href="/components/badge/badge.html">Badge</a></li>
+            <li><a href="/components/button/button.html">Button</a></li>
+            <li><a href="/components/card/card.html">Card</a></li>
+            <li><a href="/components/image/image.html">Image</a></li>
+            <li><a href="/components/input/input.html">Input</a></li>
+            <li><a href="/components/typography/typography.html">Typography</a></li>
+            <li><a href="/components/list/list.html">List</a></li>
+            <li><a href="/components/navigation/navigation.html">Navigation</a></li>
+            <li><a href="/components/modal/modal.html">Modal</a></li>
+            <!-- <li><a href="#">Rating</a></li>
+            <li><a href="#">Tost</a></li>
+            <li><a href="#">Grid</a></li>
+            <li><a href="#">Slider</a></li> -->
```

Comment:
uncomment this code and add the related links to the href's

Reply:
Hey, Husain. I have to create those but not doing them right now because of deadline constraints. Just focusing bare minimum for now.